src/libs/autoScrollback/index.web.js

```diff
@@ -0,0 +1,88 @@
+import smoothscroll from 'smoothscroll-polyfill';
+
+import getBrowser from '../getBrowser';
+
+const init = () => {
```

Comment:
We _definitely_ want to include a high-level code comment to explain what this library is for.